libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.spec.ts

```diff
@@ -134,24 +134,61 @@ describe('Summary Action Bar component', () => {
         fixture.detectChanges();
         const actionBarHeight = getActionBarHeight(debugElement);
         expect(document.body.style.marginBottom).toBe(
-          `calc(${actionBarHeight}px + var(--sky-dock-height, 0))`,
+          `calc(${actionBarHeight}px + var(--sky-dock-height, 0px))`,
```

Comment:
This just copies the string value from the adapter service to the unit test. That wasn't adequate to catch the bug originally, so we need a different approach here. I suggest hardcoding a value that you know to be correct and use `getComputedStyle()` to verify the calc function provides the expected value.

Reply:
I did that - its right below the line that you left the comment on.